src/util.ts

```diff
@@ -304,6 +304,14 @@ export async function screenshot(client: Client): Promise<string> {
   return screenshot.data
 }
 
+export async function pdf(client: Client): Promise<string> {
+  const {Page} = client
+
+  const pdf = await Page.printToPDF()
```

Comment:
I actually tried to do that before I opened this PR, and I couldn't get it to work. Even when I hard-coded `landscape: true` here, it would save a portrait PDF.

Try changing line 310 to the following:
```
const pdf = await Page.printToPDF({landscape: true})
```

It does not appear to change the output. I tried messing around with options such as `scale` and `paperHeight`, and that did not appear to affect anything either.

I'm not quite sure why passing in an options object has no effect here.